config/Config-build.in

```diff
@@ -7,6 +7,13 @@
 
 menu "Global build settings"
 
+	config JSON_ADD_IMAGE_INFO
+		bool "Create JSON info files per build image"
```

Comment:
Should that be `per built image` ?

Reply:
Yes, actually even `files per build profile`, as a profile may contain multiple images (factory, sysupgrade,...). Will update that